examples/cloudaccounts/k8s/README.md

```diff
@@ -0,0 +1,67 @@
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/dome9/dome9-sdk-go/dome9"
+	"github.com/dome9/dome9-sdk-go/services/cloudaccounts"
+	"github.com/dome9/dome9-sdk-go/services/cloudaccounts/k8s"
+)
+
+func main() {
+	// Pass accessID, secretKey, rawUrl, or set environment variables
+	config, _ := dome9.NewConfig("", "", "")
+	srv := k8s.New(config)
+	var createAccountReqBody k8s.CloudAccountRequest
+	
+	createAccountReqBody.Name = "ACCOUNT NAME" //mandatory field
+	//createAccountReqBody.OrganizationalUnitID = '11111111-2222-3333-4444-555555555555' // optional field, if not set the root OU will be set as default 
+
+	v, _, err := srv.Create(createAccountReqBody)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Printf("Response type: %T\n Content %+v\n", v, v)
+
+	// Get specific k8s cloud account
+	k8sCloudAccount, _, err := srv.Get("ACCOUNT UUID")
+	if err != nil {
+		panic(err)
+	}
+	fmt.Printf("Response type: %T\n Content: %+v\n", k8sCloudAccount, k8sCloudAccount)
+	
+	// update k8s cloud account name
+	id := "THE ACCOUNT ID"
```

Comment:
the same cluster name and the same below cluster name instead of cloud account name